app/perfsymboltable.cpp

```diff
@@ -724,7 +724,20 @@ static QByteArray fakeSymbolFromSection(Dwfl_Module *mod, Dwarf_Addr addr)
     return sym;
 }
 
-static PerfAddressCache::SymbolCache cacheSymbols(Dwfl_Module *module, quint64 elfStart)
+static void relocatedAdjust(Elf *elfp, GElf_Word shndx, quint64 elfStart, bool &relocated, quint64 &adjust) {
+    //The module is relocated
```

Comment:
space after //

also, please rephrase the comment. Is the module relocated already and this function must be only called for relocated ones? No, right? Rather:

// Check if the module is relocated